src/extractcode/archive.py

```diff
@@ -36,6 +36,7 @@
 from commoncode import functional
 from commoncode.system import on_linux
 from commoncode.system import py2
+from commoncode.ignore import is_ignored
```

Comment:
Sorry to be nit picking but could you keep this import sorted? ignore comes before system